code/modules/awaymissions/corpse.dm

```diff
@@ -40,6 +40,30 @@
 	M.timeofdeath = timeofdeath
 	if(src.mob_species)
 		M.set_species(src.mob_species)
+	if(ishuman(M))

```

Comment:
This entire check and the typecasting is redundant. Since you're spawning a `/mob/living/carbon/human/human`, it is *always* going to be human. Furthermore you don't need to typecast either.